my_list.rb

```diff
@@ -0,0 +1,39 @@
+require './my_enumerable'
```

Comment:
- [ ] Please use  `require_relative` to import "my_enumerable" for files instead of `require` because `Require` should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory."

Reply:
Well resolved 👍🏽